packages/main/src/plugin/color-registry.ts

```diff
@@ -808,4 +808,86 @@ export class ColorRegistry {
       light: colorPalette.black + '2',
     });
   }
+
+  // button
+  protected initButton(): void {
+    const button = 'button-';
+
+    this.registerColor(`${button}primary-bg`, {
+      dark: colorPalette.purple[600],
+      light: colorPalette.purple[500],
```

Comment:
light: colorPalette.purple[600]

Reply:
Hey! Github has a feature where you can make a `suggestion` :) might be useful :)

Reply:
oh cool I'll look into it! Thanks :)

Reply:
Here is a short demo if that can help :)

https://github.com/containers/podman-desktop/assets/42176370/e772f762-d549-46df-9837-706227d3920e

Reply:
Original suggestion merged.